lib/common_test/src/ct_snmp.erl

```diff
@@ -241,43 +254,34 @@ stop(Config) ->
     catch del_dir(MgrDir),
     catch del_dir(ConfDir),
     catch del_dir(DbDir).
-    
-    
-%%% @spec get_values(Agent, Oids, MgrAgentConfName) -> SnmpReply
-%%%
-%%%	 Agent = agent_name()
-%%%      Oids = oids()
-%%%      MgrAgentConfName = atom()
-%%%      SnmpReply = snmpreply()  
-%%%
-%%% @doc Issues a synchronous snmp get request. 
+
+
+%%% @doc Issues a synchronous snmp get request.
+-spec get_values(Agent, Oids, MgrAgentConfName) -> snmpreply() when
+      Agent :: agent_name(),
+      Oids :: oids(),
+      MgrAgentConfName :: atom().
 get_values(Agent, Oids, MgrAgentConfName) ->
     [Uid | _] = agent_conf(Agent, MgrAgentConfName),
     {ok, SnmpReply, _} = snmpm:sync_get2(Uid, target_name(Agent), Oids),
     SnmpReply.
 
-%%% @spec get_next_values(Agent, Oids, MgrAgentConfName) -> SnmpReply 
-%%%
-%%%	 Agent = agent_name()
-%%%      Oids = oids()
-%%%      MgrAgentConfName = atom()
-%%%      SnmpReply = snmpreply()  
-%%%
-%%% @doc Issues a synchronous snmp get next request. 
+%%% @doc Issues a synchronous snmp get next request.
+-spec get_next_values(Agent, Oids, MgrAgentConfName) -> snmpreply() when
+      Agent :: agent_name(),
+      Oids :: oids(),
+      MgrAgentConfName :: atom().
 get_next_values(Agent, Oids, MgrAgentConfName) ->
     [Uid | _] = agent_conf(Agent, MgrAgentConfName),
     {ok, SnmpReply, _} = snmpm:sync_get_next2(Uid, target_name(Agent), Oids),
     SnmpReply.
 
-%%% @spec set_values(Agent, VarsAndVals, MgrAgentConfName, Config) -> SnmpReply
-%%%
-%%%	 Agent = agent_name()
-%%%      Oids = oids()
-%%%      MgrAgentConfName = atom()
-%%%      Config = [{Key, Value}] 
-%%%      SnmpReply = snmpreply()  
-%%%
-%%% @doc Issues a synchronous snmp set request. 
+%%% @doc Issues a synchronous snmp set request.
+-spec set_values(Agent, VarsAndVals, MgrAgentConfName, Config) -> snmpreply() when
+      Agent :: agent_name(),
+      VarsAndVals :: vars_and_vals(),
+      MgrAgentConfName :: atom(),
+      Config :: agent_config().
```

Comment:
This should not be agent_config(). It is the Config variable from the test case, defined as [{Key,Value}], where Key::atom(), Value::term().